mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,20 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(isHappy) {
+  //let isHappy = true;
 
-  if (isHappy) {
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough=num;
```

Comment:
Looks good but I'm wondering whether you can make it better? You did similar changes already in the `getMood` function. Would that work here too?